A43-sdk-authorization.md

```diff
@@ -276,40 +283,26 @@ In the following policy example
 }
 ```
 
-### High level Implementation
-
-gRPC Authorization internally implements RBAC Engines based on Envoy RBAC policies.
-RBAC policy provides service-level and method-level access control for a service.
-Engines process incoming RPC request attributes against policy configs and make a
-decision on whether to allow or deny the request. The decision depends on the type
-of policy (if the policy is an allowlist or denylist) and whether a matching policy
-was found. Engine implementation is shared by xDS and SDK authorization.
-
-[RBAC filter]: https://github.com/envoyproxy/envoy/blob/main/api/envoy/extensions/filters/http/rbac/v3/rbac.proto
-
-In SDK authorization, user supplies gRPC SDK authorization policy to policy provider
-interface. In the case of file watcher, the provider is responsible for initializing
-the thread(C++)/ goroutine(Go)/ scheduled service(Java) which will be used to read
-the policy file periodically. The provider then forwards the JSON policy to Policy
-translator. The translator converts JSON policy to Envoy RBAC protos (Allow and/or
-Deny policy). Translator errors out on I/O errors or if the policy does not
-represent JSON schema it currently supports.
-
-Note that SDK authorization policy is a subset of Envoy RBAC, and it does not
-support all the fields that are present in Envoy RBAC. Ultimately the generated
-RBAC policies are used to create Envoy RBAC engine(s).
-
-For each incoming RPC request, we will invoke the Evaluate functionality in Engines
-(Deny engine followed by Allow engine), to get the authorization decision. We use a
-C-core filter for C++, and interceptors for Java and Go.
-
 ### API
 
 gRPC will support both static initialization and dynamically reloading the policy
 from filesystem. In static initialization, the policy will be provided as a JSON
 string. In dynamic file reloading, the application will specify the file path that
 contains the authorization policy in JSON format.
 
+Valid user provided SDK authorization policy creates authorization engine(s).
+In the case of file watcher, we internally create thread(C-core)/ goroutine(Go)/
+scheduled service(Java) which will be used to read the policy file periodically,
+and update the authorization engines. During the first file read, if the policy
+is invalid or there are I/O errrors, we will return error back to application and
+SDK authorization won't be enabled. If the error occurs on a later reload, then
```

Comment:
Saying that SDK authz won't be enabled implies that the server will start with no authz policy protection at all, which is not what we want.  This should probably just say something like "so that the gRPC server will not start".

Reply:
I removed the part "and SDK authorization won't be enabled". PTAL